ci/jenkins/jobs/macros.yaml

```diff
@@ -223,3 +223,37 @@
           set -e
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
           ./ci/jenkins/test-vm.sh  --registry ${DOCKER_REGISTRY} --kubeconfig /var/lib/jenkins/.kube/config
+          
+- builder:
+    name: builder-e2e-kind
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/kind/kind-setup.sh --antrea-cni create "${JOB_NAME}-${BUILD_NUMBER}"
+          kind export kubeconfig -n "${JOB_NAME}-${BUILD_NUMBER}" --kubeconfig ${PWD}/.kube/config
+          set +ex
+          ./ci/jenkins/test.sh --testcase e2e --registry ${DOCKER_REGISTRY} --kubeconfig ${PWD}/.kube/config --testbed-type "kind" --kind-cluster-name "${JOB_NAME}-${BUILD_NUMBER}"
+          return_code=$?
+          set -ex
+          kind delete cluster --name "${JOB_NAME}-${BUILD_NUMBER}"
+          exit $return_code
+          
+
+- builder:
+    name: builder-conformance-kind
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          ./ci/kind/kind-setup.sh --antrea-cni create "${{JOB_NAME}}-${{BUILD_NUMBER}}"
```

Comment:
The github workflow can also run kind test. I suggest to run kind in Jenkins because we can create larger VM as the Jenkins node, so we can run conformance test which requires more resource.

I think we should try to run conformance in github workflow once to confirm this: github workflow runner doesn't offer sufficient resources (memory, CPU), so conformance test will fail in it.

Reply:
But also, we already have a lot of Github workflows and it can create some long queuing times based on worker availability.